zrml/swaps/src/tests.rs

```diff
@@ -1434,6 +1507,39 @@ fn swap_exact_amount_out_exchanges_correct_values_with_cpmm() {
     });
 }
 
+#[test]
+fn swap_exact_amount_out_fails_if_min_asset_amount_out_is_not_satisfied_with_cpmm() {
+    ExtBuilder::default().build().execute_with(|| {
+        create_initial_pool_with_funds_for_alice(ScoringRule::CPMM, true);
+        let expected_amount = 10101010100;
+        assert_noop!(
+            Swaps::swap_exact_amount_out(
+                alice_signed(),
+                0,
+                ASSET_A,
+                Some(expected_amount - 100),
+                ASSET_B,
+                _1,
+                None,
+            ),
+            crate::Error::<Runtime>::LimitIn,
+        );
+    });
+}
```

Comment:
Same as mentioned above.

Reply:
Mitigated in 94ed576d8fb6ecf38bd50fd48d206b211bb39be9